transit/world_feed/world_feed_tests/world_feed_tests.cpp

```diff
@@ -311,25 +311,25 @@ UNIT_TEST(Transit_GTFS_ProjectStopToLine_MaxDistance)
                                           Direction::Forward, shape), ());
 }
 
-// Complex shape with multiple points on it and multiple stops for projection.
-//
-//                   +-----+
-//              C*  /       \
-//          /+\    /         \  *D
-//       + /    \+/           \
-//      /                     +
-//     /                      |       *E
-//    +                       +-----+
-//    |                             |
-//    |                             |
-//    +---+\                  +-----+
-//          \                 |
-//       B*  +                |
-//       A*   \     +---------+
-//             +    |
-//             |    +
-//             +         *F
-//
+/* Complex shape with multiple points on it and multiple stops for projection.
+ *
+ *                   +-----+
+ *              C*  /       \
+ *          /+\    /         \  *D
+ *       + /    \+/           \
+ *      /                     +
+ *     /                      |       *E
+ *    +                       +-----+
+ *    |                             |
+ *    |                             |
+ *    +---+\                  +-----+
+ *          \                 |
+ *       B*  +                |
+ *       A*   \     +---------+
+ *             +    |
+ *             |    +
+ *             +         *F
+ */
```

Comment:
What is the reason here?

Reply:
```cpp
//              C*  /       \ 


//       + /    \+/           \
```

problem in `\` symbols at the end of the line
multi-line comment [-Wcomment]